pkg/hooks/manager.go

```diff
@@ -56,6 +56,12 @@ type callBackClient struct {
 var manager Manager
 var once sync.Once
 
+var SupportedVersions = map[string]bool{
```

Comment:
nit: I think the comment is still relevant here:
```suggestion
// The order matters. We should match newer versions first.
var SupportedVersions = map[string]bool{
```